playbooks/common/openshift-cluster/disable_excluder.yml

```diff
@@ -3,9 +3,29 @@
   hosts: l_oo_all_hosts
   gather_facts: no
   tasks:
+
+  # During installation the excluders are installed with present state.
+  # So no pre-validation check here as the excluders are either to be installed (present = latest)
+  # or they are not going to be updated if already installed
+
+  # Install any excluder that is enabled
+  - include_role:
+      name: openshift_excluder
+      tasks_from: init
+  - include_role:
+      name: openshift_excluder
+      tasks_from: install
+    vars:
+      # Both docker_excluder_on and openshift_excluder_on are set in openshift_excluder->init task
+      install_docker_excluder: docker_excluder_on
+      install_openshift_excluder: openshift_excluder_on
+
+  # Check the current state of all excluders
   - include_role:
       name: openshift_excluder
       tasks_from: status
+
+  # And finally adjust an excluder in order to update host components correctly
   - include_role:
       name: openshift_excluder
-      tasks_from: unexclude
+      tasks_from: adjust
```

Comment:
I have introduce new enable/disable tasks that should be general and configurable enough to cover the case.